Cargo.toml

```diff
@@ -13,7 +13,7 @@ clap = { version = "4", features = ["derive"] }
 dirs = "4.0"
 educe = { version = "0.4", features = ["Debug", "Default"] }
 ethereum-interfaces = { git = "https://github.com/ledgerwatch/interfaces" }
-ethers = "1.0.0"
+ethers = { git = "https://github.com/gakonst/ethers-rs", branch = "master", features = ["ethers-solc"]  }
```

Comment:
branch = "master" is not stable because master is always changing. We should use some rev directly until they release a new version.